scripts/ci/kubernetes/app/templates/airflow.template.yaml

```diff
@@ -46,31 +46,53 @@ spec:
         runAsGroup: 50000
         fsGroup: 50000
       initContainers:
-      - name: "init"
-        image: {{AIRFLOW_KUBERNETES_IMAGE}}
-        imagePullPolicy: IfNotPresent
-        volumeMounts:
-        - name: airflow-configmap
-          mountPath: /opt/airflow/airflow.cfg
-          subPath: airflow.cfg
-        - name: {{INIT_DAGS_VOLUME_NAME}}
-          mountPath: /opt/airflow/dags
-        env:
-        - name: SQL_ALCHEMY_CONN
-          valueFrom:
-            secretKeyRef:
-              name: airflow-secrets
-              key: sql_alchemy_conn
-        command:
-          - "bash"
-        args:
-          - "-cx"
-          - "/tmp/airflow-test-env-init.sh"
+        - name: "init-dags"
```

Comment:
good idea! Should make things move faster

Reply:
A bit. I hope we will replace those templates with Helm charts eventually and we get rid of them. Those are a bit abominations.

Reply:
100% agreed.

Reply:
I regret not setting up a helm chart from the start, but at the time I was at Bloomberg and Bloomberg was heavily of the opinion that helm was unsafe/bad.